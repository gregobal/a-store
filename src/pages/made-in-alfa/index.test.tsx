import {render} from "@testing-library/react";
import {MemoryRouter} from "react-router-dom";
import {MadeInAlfaPage} from "./index";

describe("Made In Alfa Page", () => {
    it("snapshot", () => {
        const view = render(
            <MemoryRouter>
                <MadeInAlfaPage />
            </MemoryRouter>
        );

        expect(view).toMatchSnapshot();
    });
});
