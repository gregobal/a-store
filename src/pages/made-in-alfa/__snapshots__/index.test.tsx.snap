// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Made In Alfa Page snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <h1
        class="typography__component_h1nzy typography__styrene-xlarge_qohs3 typography__bold_h1nzy"
      >
        Сделано в Альфе
      </h1>
      <span
        class="typography__primary-medium_xcdcz typography__bold_xcdcz"
      >
        Хотим каждую из этих вещей! Себе, родным и друзьям
      </span>
      <div
        class="gap__gap_t9kgt gap__vertical_t9kgt"
        data-gap-size="m"
      />
      <article
        class="grid__row_1gt5e grid__component_xpkmx grid__left_xpkmx grid__gutter-24_1gt5e"
      >
        <section
          class="grid__col_1gt5e grid__component_17rb3 grid__width-mobile-12_17rb3 grid__width-tablet-6_17rb3 grid__width-desktop-4_17rb3"
        >
          <a
            class="pure-cell__link_167we pure-cell__component_167we pure-cell__vertical_167we pure-cell__airy_167we pure-cell__none_167we"
            href="/sdelano-v-alfe/0"
          >
            <div
              class="skeleton__component_jp9h2 skeleton__animate_jp9h2 wrapper w1h1"
            >
              <img
                alt="Рюкзак «Для умных и свободных»"
                class="image"
                src="http://qa-games.ru/astore/public/images/45862782.jpeg"
                width="368"
              />
            </div>
            <div
              class="gap__gap_t9kgt gap__vertical_t9kgt"
              data-gap-size="xs"
            />
            <section
              class="pure-cell__component_1ymlk"
            >
              <div
                class="pure-cell__component_1ng4h pure-cell__vertical_1ng4h"
              >
                <h2
                  class="typography__component_h1nzy typography__styrene-xsmall_qohs3 typography__medium_h1nzy"
                >
                  Рюкзак «Для умных и свободных»
                </h2>
                <span
                  class="pure-cell__component_jmf8b typography__primary_1faea typography__component_xcdcz"
                >
                  <span
                    class="amount__component_96zmx amount__bold_96zmx"
                  >
                    4 999
                    <span
                      class=""
                    >
                       ₽
                    </span>
                  </span>
                </span>
              </div>
            </section>
          </a>
        </section>
        <section
          class="grid__col_1gt5e grid__component_17rb3 grid__width-mobile-12_17rb3 grid__width-tablet-6_17rb3 grid__width-desktop-4_17rb3"
        >
          <a
            class="pure-cell__link_167we pure-cell__component_167we pure-cell__vertical_167we pure-cell__airy_167we pure-cell__none_167we"
            href="/sdelano-v-alfe/1"
          >
            <div
              class="skeleton__component_jp9h2 skeleton__animate_jp9h2 wrapper w1h1"
            >
              <img
                alt="Футболка Для умных и свободных"
                class="image"
                src="http://qa-games.ru/astore/public/images/68519498.jpeg"
                width="368"
              />
            </div>
            <div
              class="gap__gap_t9kgt gap__vertical_t9kgt"
              data-gap-size="xs"
            />
            <section
              class="pure-cell__component_1ymlk"
            >
              <div
                class="pure-cell__component_1ng4h pure-cell__vertical_1ng4h"
              >
                <h2
                  class="typography__component_h1nzy typography__styrene-xsmall_qohs3 typography__medium_h1nzy"
                >
                  Футболка Для умных и свободных
                </h2>
                <span
                  class="pure-cell__component_jmf8b typography__primary_1faea typography__component_xcdcz"
                >
                  <span
                    class="amount__component_96zmx amount__bold_96zmx"
                  >
                    1 999
                    <span
                      class=""
                    >
                       ₽
                    </span>
                  </span>
                </span>
              </div>
            </section>
          </a>
        </section>
        <section
          class="grid__col_1gt5e grid__component_17rb3 grid__width-mobile-12_17rb3 grid__width-tablet-6_17rb3 grid__width-desktop-4_17rb3"
        >
          <a
            class="pure-cell__link_167we pure-cell__component_167we pure-cell__vertical_167we pure-cell__airy_167we pure-cell__none_167we"
            href="/sdelano-v-alfe/2"
          >
            <div
              class="skeleton__component_jp9h2 skeleton__animate_jp9h2 wrapper w1h1"
            >
              <img
                alt="Блокнот Для умных и свободных"
                class="image"
                src="http://qa-games.ru/astore/public/images/77117755.jpeg"
                width="368"
              />
            </div>
            <div
              class="gap__gap_t9kgt gap__vertical_t9kgt"
              data-gap-size="xs"
            />
            <section
              class="pure-cell__component_1ymlk"
            >
              <div
                class="pure-cell__component_1ng4h pure-cell__vertical_1ng4h"
              >
                <h2
                  class="typography__component_h1nzy typography__styrene-xsmall_qohs3 typography__medium_h1nzy"
                >
                  Блокнот Для умных и свободных
                </h2>
                <span
                  class="pure-cell__component_jmf8b typography__primary_1faea typography__component_xcdcz"
                >
                  <span
                    class="amount__component_96zmx amount__bold_96zmx"
                  >
                    1 499
                    <span
                      class=""
                    >
                       ₽
                    </span>
                  </span>
                </span>
              </div>
            </section>
          </a>
        </section>
        <section
          class="grid__col_1gt5e grid__component_17rb3 grid__width-mobile-12_17rb3 grid__width-tablet-6_17rb3 grid__width-desktop-4_17rb3"
        >
          <a
            class="pure-cell__link_167we pure-cell__component_167we pure-cell__vertical_167we pure-cell__airy_167we pure-cell__none_167we"
            href="/sdelano-v-alfe/3"
          >
            <div
              class="skeleton__component_jp9h2 skeleton__animate_jp9h2 wrapper w1h1"
            >
              <img
                alt="Чехол с кардхолдером"
                class="image"
                src="http://qa-games.ru/astore/public/images/15932051.jpeg"
                width="368"
              />
            </div>
            <div
              class="gap__gap_t9kgt gap__vertical_t9kgt"
              data-gap-size="xs"
            />
            <section
              class="pure-cell__component_1ymlk"
            >
              <div
                class="pure-cell__component_1ng4h pure-cell__vertical_1ng4h"
              >
                <h2
                  class="typography__component_h1nzy typography__styrene-xsmall_qohs3 typography__medium_h1nzy"
                >
                  Чехол с кардхолдером
                </h2>
                <span
                  class="pure-cell__component_jmf8b typography__disabled_1faea typography__component_xcdcz"
                >
                  <span
                    class="amount__component_96zmx amount__bold_96zmx"
                  >
                    799
                    <span
                      class=""
                    >
                       ₽
                    </span>
                  </span>
                </span>
              </div>
            </section>
          </a>
        </section>
        <section
          class="grid__col_1gt5e grid__component_17rb3 grid__width-mobile-12_17rb3 grid__width-tablet-6_17rb3 grid__width-desktop-4_17rb3"
        >
          <a
            class="pure-cell__link_167we pure-cell__component_167we pure-cell__vertical_167we pure-cell__airy_167we pure-cell__none_167we"
            href="/sdelano-v-alfe/4"
          >
            <div
              class="skeleton__component_jp9h2 skeleton__animate_jp9h2 wrapper w1h1"
            >
              <img
                alt="Экоручка"
                class="image"
                src="http://qa-games.ru/astore/public/images/56369345.jpeg"
                width="368"
              />
            </div>
            <div
              class="gap__gap_t9kgt gap__vertical_t9kgt"
              data-gap-size="xs"
            />
            <section
              class="pure-cell__component_1ymlk"
            >
              <div
                class="pure-cell__component_1ng4h pure-cell__vertical_1ng4h"
              >
                <h2
                  class="typography__component_h1nzy typography__styrene-xsmall_qohs3 typography__medium_h1nzy"
                >
                  Экоручка
                </h2>
                <span
                  class="pure-cell__component_jmf8b typography__primary_1faea typography__component_xcdcz"
                >
                  <span
                    class="amount__component_96zmx amount__bold_96zmx"
                  >
                    99
                    <span
                      class=""
                    >
                       ₽
                    </span>
                  </span>
                </span>
              </div>
            </section>
          </a>
        </section>
      </article>
    </div>
  </body>,
  "container": <div>
    <h1
      class="typography__component_h1nzy typography__styrene-xlarge_qohs3 typography__bold_h1nzy"
    >
      Сделано в Альфе
    </h1>
    <span
      class="typography__primary-medium_xcdcz typography__bold_xcdcz"
    >
      Хотим каждую из этих вещей! Себе, родным и друзьям
    </span>
    <div
      class="gap__gap_t9kgt gap__vertical_t9kgt"
      data-gap-size="m"
    />
    <article
      class="grid__row_1gt5e grid__component_xpkmx grid__left_xpkmx grid__gutter-24_1gt5e"
    >
      <section
        class="grid__col_1gt5e grid__component_17rb3 grid__width-mobile-12_17rb3 grid__width-tablet-6_17rb3 grid__width-desktop-4_17rb3"
      >
        <a
          class="pure-cell__link_167we pure-cell__component_167we pure-cell__vertical_167we pure-cell__airy_167we pure-cell__none_167we"
          href="/sdelano-v-alfe/0"
        >
          <div
            class="skeleton__component_jp9h2 skeleton__animate_jp9h2 wrapper w1h1"
          >
            <img
              alt="Рюкзак «Для умных и свободных»"
              class="image"
              src="http://qa-games.ru/astore/public/images/45862782.jpeg"
              width="368"
            />
          </div>
          <div
            class="gap__gap_t9kgt gap__vertical_t9kgt"
            data-gap-size="xs"
          />
          <section
            class="pure-cell__component_1ymlk"
          >
            <div
              class="pure-cell__component_1ng4h pure-cell__vertical_1ng4h"
            >
              <h2
                class="typography__component_h1nzy typography__styrene-xsmall_qohs3 typography__medium_h1nzy"
              >
                Рюкзак «Для умных и свободных»
              </h2>
              <span
                class="pure-cell__component_jmf8b typography__primary_1faea typography__component_xcdcz"
              >
                <span
                  class="amount__component_96zmx amount__bold_96zmx"
                >
                  4 999
                  <span
                    class=""
                  >
                     ₽
                  </span>
                </span>
              </span>
            </div>
          </section>
        </a>
      </section>
      <section
        class="grid__col_1gt5e grid__component_17rb3 grid__width-mobile-12_17rb3 grid__width-tablet-6_17rb3 grid__width-desktop-4_17rb3"
      >
        <a
          class="pure-cell__link_167we pure-cell__component_167we pure-cell__vertical_167we pure-cell__airy_167we pure-cell__none_167we"
          href="/sdelano-v-alfe/1"
        >
          <div
            class="skeleton__component_jp9h2 skeleton__animate_jp9h2 wrapper w1h1"
          >
            <img
              alt="Футболка Для умных и свободных"
              class="image"
              src="http://qa-games.ru/astore/public/images/68519498.jpeg"
              width="368"
            />
          </div>
          <div
            class="gap__gap_t9kgt gap__vertical_t9kgt"
            data-gap-size="xs"
          />
          <section
            class="pure-cell__component_1ymlk"
          >
            <div
              class="pure-cell__component_1ng4h pure-cell__vertical_1ng4h"
            >
              <h2
                class="typography__component_h1nzy typography__styrene-xsmall_qohs3 typography__medium_h1nzy"
              >
                Футболка Для умных и свободных
              </h2>
              <span
                class="pure-cell__component_jmf8b typography__primary_1faea typography__component_xcdcz"
              >
                <span
                  class="amount__component_96zmx amount__bold_96zmx"
                >
                  1 999
                  <span
                    class=""
                  >
                     ₽
                  </span>
                </span>
              </span>
            </div>
          </section>
        </a>
      </section>
      <section
        class="grid__col_1gt5e grid__component_17rb3 grid__width-mobile-12_17rb3 grid__width-tablet-6_17rb3 grid__width-desktop-4_17rb3"
      >
        <a
          class="pure-cell__link_167we pure-cell__component_167we pure-cell__vertical_167we pure-cell__airy_167we pure-cell__none_167we"
          href="/sdelano-v-alfe/2"
        >
          <div
            class="skeleton__component_jp9h2 skeleton__animate_jp9h2 wrapper w1h1"
          >
            <img
              alt="Блокнот Для умных и свободных"
              class="image"
              src="http://qa-games.ru/astore/public/images/77117755.jpeg"
              width="368"
            />
          </div>
          <div
            class="gap__gap_t9kgt gap__vertical_t9kgt"
            data-gap-size="xs"
          />
          <section
            class="pure-cell__component_1ymlk"
          >
            <div
              class="pure-cell__component_1ng4h pure-cell__vertical_1ng4h"
            >
              <h2
                class="typography__component_h1nzy typography__styrene-xsmall_qohs3 typography__medium_h1nzy"
              >
                Блокнот Для умных и свободных
              </h2>
              <span
                class="pure-cell__component_jmf8b typography__primary_1faea typography__component_xcdcz"
              >
                <span
                  class="amount__component_96zmx amount__bold_96zmx"
                >
                  1 499
                  <span
                    class=""
                  >
                     ₽
                  </span>
                </span>
              </span>
            </div>
          </section>
        </a>
      </section>
      <section
        class="grid__col_1gt5e grid__component_17rb3 grid__width-mobile-12_17rb3 grid__width-tablet-6_17rb3 grid__width-desktop-4_17rb3"
      >
        <a
          class="pure-cell__link_167we pure-cell__component_167we pure-cell__vertical_167we pure-cell__airy_167we pure-cell__none_167we"
          href="/sdelano-v-alfe/3"
        >
          <div
            class="skeleton__component_jp9h2 skeleton__animate_jp9h2 wrapper w1h1"
          >
            <img
              alt="Чехол с кардхолдером"
              class="image"
              src="http://qa-games.ru/astore/public/images/15932051.jpeg"
              width="368"
            />
          </div>
          <div
            class="gap__gap_t9kgt gap__vertical_t9kgt"
            data-gap-size="xs"
          />
          <section
            class="pure-cell__component_1ymlk"
          >
            <div
              class="pure-cell__component_1ng4h pure-cell__vertical_1ng4h"
            >
              <h2
                class="typography__component_h1nzy typography__styrene-xsmall_qohs3 typography__medium_h1nzy"
              >
                Чехол с кардхолдером
              </h2>
              <span
                class="pure-cell__component_jmf8b typography__disabled_1faea typography__component_xcdcz"
              >
                <span
                  class="amount__component_96zmx amount__bold_96zmx"
                >
                  799
                  <span
                    class=""
                  >
                     ₽
                  </span>
                </span>
              </span>
            </div>
          </section>
        </a>
      </section>
      <section
        class="grid__col_1gt5e grid__component_17rb3 grid__width-mobile-12_17rb3 grid__width-tablet-6_17rb3 grid__width-desktop-4_17rb3"
      >
        <a
          class="pure-cell__link_167we pure-cell__component_167we pure-cell__vertical_167we pure-cell__airy_167we pure-cell__none_167we"
          href="/sdelano-v-alfe/4"
        >
          <div
            class="skeleton__component_jp9h2 skeleton__animate_jp9h2 wrapper w1h1"
          >
            <img
              alt="Экоручка"
              class="image"
              src="http://qa-games.ru/astore/public/images/56369345.jpeg"
              width="368"
            />
          </div>
          <div
            class="gap__gap_t9kgt gap__vertical_t9kgt"
            data-gap-size="xs"
          />
          <section
            class="pure-cell__component_1ymlk"
          >
            <div
              class="pure-cell__component_1ng4h pure-cell__vertical_1ng4h"
            >
              <h2
                class="typography__component_h1nzy typography__styrene-xsmall_qohs3 typography__medium_h1nzy"
              >
                Экоручка
              </h2>
              <span
                class="pure-cell__component_jmf8b typography__primary_1faea typography__component_xcdcz"
              >
                <span
                  class="amount__component_96zmx amount__bold_96zmx"
                >
                  99
                  <span
                    class=""
                  >
                     ₽
                  </span>
                </span>
              </span>
            </div>
          </section>
        </a>
      </section>
    </article>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
